test/cli/fixtures/expected/tap-outputs.js

```diff
@@ -126,15 +126,21 @@ not ok 2 global failure
 # fail 2
 `,
 
+	// in node 8, the stack trace includes 'at <anonymous>. But not in node 6 or 10.
 	"qunit no-tests":
 `TAP version 13
 not ok 1 global failure
   ---
   message: "No tests were run."
   severity: failed
-  actual: null
+  actual: {}
   expected: undefined
-  stack: undefined:undefined
+  stack: Error: No tests were run.
+    at done (.*)
```

Comment:
Should the parentheses on this and consequent lines be escaped?

Reply:
good call, it should be!